pages/linux/tcpick.md

```diff
@@ -0,0 +1,17 @@
+# tcpick
+
+> A Linux command-line tool for packet sniffing and network traffic analysis.
+> It can capture and display TCP connections and data. It can also monitor network traffic on a specific interface, host, or port.
+> More information: <https://linux.die.net/man/8/tcpick>.
```

Comment:
```suggestion
> Packet sniffing and network traffic analysis tool.
> It can capture and display TCP connections and data. It can also monitor network traffic on a specific interface, host, or port.
> More information: <https://manned.org/tcpick.8>.
```
Regarding the page description, we prefer not to mention the platform/CLI tool as it might seem redundant and as for the more information link we fall back to manned when there isn't an official documentation (manpage).